examples/neighbors/plot_lmnn_illustration.py

```diff
@@ -0,0 +1,153 @@
+"""
+==========================================
+Large Margin Nearest Neighbor Illustration
+==========================================
+
+An example illustrating the goal of learning a distance metric that maximizes
```

Comment:
What about
"This example illustrates...."

Reply:
Done.